maxwell/sed.py

```diff
@@ -174,7 +174,7 @@ def build_sed(
     @classmethod
     def fit_from_data(
         cls,
-        lines: Iterable[Tuple[Any]],
+        lines: Iterable[Tuple[str, str]],
```

Comment:
Should still be any since this also permits int/float

Reply:
You're totally right. I added a brief note to the docstring that it should work with anything hashable.